cypress/integration/seededFlows/notificationsFlows/followUserFromNotifications.spec.js

```diff
@@ -10,16 +10,16 @@ describe('Follow user from notifications', () => {
 
   it('Follows and unfollows a user from a follow notification', () => {
     cy.findByRole('heading', { name: 'Notifications' });
-    cy.intercept('/follows').as('followsRequest');
+    // cy.intercept('/follows').as('followsRequest');
```

Comment:
I can't recall why we had this intercept here, but if we don't need it any more, go ahead and delete 🙂 
```suggestion
```